sd-rsyslog

```diff
@@ -0,0 +1,193 @@
+#!/opt/venvs/securedrop-log/bin/python3
+"""A skeleton for a Python rsyslog output plugin with error handling.
+Requires Python 3.
+
+To integrate a plugin based on this skeleton with rsyslog, configure an
+'omprog' action like the following:
+    action(type="omprog"
+        binary="/usr/bin/myplugin.py"
+        output="/var/log/myplugin.log"
+        confirmMessages="on"
+        ...)
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
should we licence (a)GPLv3 here for consistency with both other repos and your setup.py: https://github.com/freedomofpress/securedrop-log/pull/8/files#diff-2eeaed663bd0d25b7e608891384b7298R27

Reply:
We can not as this is copied from the official example.